.storybook/config.js

```diff
@@ -39,13 +39,16 @@ const prefix = (phrase, prefix) => value => (value.indexOf(phrase) > -1 ? prefix
 const pipe = (...fns) => value => fns.reduce((result, fn) => fn(result), value);
 
 function storySort(a, b) {
+  // console.warn('b', b);
   const prefixFn = pipe(
     prefix('welcome-', '0'),
     prefix('getting-started', '0'),
     prefix('tokens-', '1'),
-    prefix('components-', '2'),
+    prefix('components-', '3'),
```

Comment:
Is it necessary to change this one? This may be related to the edge cases mentioned in my other comment